_posts/2021-06-23-cockpit-247.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cockpit 247
+author: kkoukiou
+date: 2021-06-23
+tags: cockpit linux
+slug: cockpit-247
+category: release
+summary: Cockpit with Metrics and Virtual Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 247 and Cockpit Machines version 246.
+
+## Metrics: Enable Grafana client setup
+
+"Metrics settings", [introduced in Cockpit 245](https://cockpit-project.org/blog/cockpit-245.html), has a new option to share Performance Co-Pilot data over the network. Toggling "export to network" on enables a [Redis](https://grafana-pcp.readthedocs.io/en/latest/datasources/redis.html) data source with a [pmproxy service](https://linux.die.net/man/1/pmproxy) and adjusts firewall settings as needed.
```

Comment:
I find this a bit confusing now. "Toggling <blah> on enables.." is hard to read for me as non-native speaker at least. "a" pmproxy service is also confusing, there is just one. Finally, the metrics settings dialog does *not* adjust firewall settings, this is done by the following paragraph.


```suggestion
"Metrics settings", [introduced in Cockpit 245](https://cockpit-project.org/blog/cockpit-245.html), has a new option to share Performance Co-Pilot data over the network. Enabling "export to network" enables a [Redis](https://grafana-pcp.readthedocs.io/en/latest/datasources/redis.html) data source with the [pmproxy service](https://linux.die.net/man/1/pmproxy).
```